packages/repeat/src/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Repeats the input text number of times.
+ * @param {String} [text] The input text.
+ * @returns {Integer} times How many times shall we repeat the text, default to 1
+ * @param {String} glue The glue for the output, default to ''
+ * @returns {String} The repeated string;
+ */
+const repeat = (text, repeat = 1, glue = '') => {
```

Comment:
The function and the variable are both name `repeat`. 😬

Reply:
I changed param name form 'repeat' to 'times'.